app/static/js/Scanner.jsx

```diff
@@ -55,24 +57,39 @@ class Scanner extends React.Component {
   }
 
   handleDetectedBarcode = barcode => {
+    console.log(barcode);
+    const code = parseInt(barcode.codeResult.code);
+    const direction = 'IN';
     this.setState({scanning: false});
-    this.props.incrementAttendance();
+    axios.get('/api/check_ticket', {
+      params: {
+        ticket_id: code,
+        direction: direction
+      }
+    })
+    .then(response => {
+      console.log(response);
+      if (response.data.action === 1) {
+        this.props.incrementAttendance();
```

Comment:
I think we can get the time from the response, instead of doing `new Date()` in incrementAttendance.